keywords.js

```diff
@@ -434,6 +434,36 @@ const KEYWORDS = {
     opt: "tright",
     ignore_comments_after: false,
   },
+  EDIT: {
+    mandat: "e",
+    opt: "dit",
+    ignore_comments_after: false,
+  },
+  ENEW: {
+    mandat: "ene",
+    opt: "w",
+    ignore_comments_after: false,
+  },
+  FIND: {
+    mandat: "fin",
+    opt: "d",
+    ignore_comments_after: false,
+  },
+  EX: {
+    mandat: "ex",
+    opt: "",
+    ignore_comments_after: false,
+  },
+  VISUAL: {
+    mandat: "vi",
+    opt: "sual",
+    ignore_comments_after: false,
+  },
```

Comment:
I don't think those are `edit` variants, but we'll accept them anyway.

Reply:
from doc:
```
:ex [++opt] [+cmd] [file]
			Same as |:edit|.
```
So at least ex is the same.
For visual, I don't think it's possible to use it in Ex-mode in scripts, and if not in Ex-mode, it is the same as `:edit:`
Same goes for `:view`

Reply:
'kay may bad then